docs/adr/0032-l10n-i18n-improvements.md

```diff
@@ -0,0 +1,138 @@
+- `@fluent/react` docs actually [recommend fallback text](https://github.com/projectfluent/fluent.js/wiki/Localized#fallback-string) and state that in the future, it may be used for automatic [extraction of source copy](https://github.com/projectfluent/fluent.js/wiki/Localized#fallback-string)
+- We ran a small test (see technical story doc) that showed at least on rare occasions, Fluent may fail to retrieve strings
+- It would take at least two searches to find where a string should ultimately end up in a component, e.g. finding the corresponding FTL ID then searching for it
+
+### Decision outcome 2: continue using fallback text and set up better testing strategies
+
+One of the primary concerns around using fallback text is maintainability and consistency guarantee. We should be testing:
+
+- All provided FTL IDs exist in the source-of-truth `en` bundle
+- Fallback text matches the message tied to the ID in the `en.ftl` bundle, and validate that variables referenced in the message string are available
+- The message doesn’t contain straight quotes (nice to have)
+
+#### React-based packages
+
+We will create two wrappers, one for `Localized` and one for `l10n.getString()`, to enforce fallback text is always provided, set up a `merge:ftl-test` script for when tests run, and mock the wrappers to test what's outlined above.
+
+#### `fxa-auth-server`
+
+We will create a helper function to send into our templates to more easily grab messages from Fluent bundles for things like alt text that otherwise require additional complexities due to our MJML and `@fluent/dom` set up. We can also automatically apply `data-l10n-args`.
+
+We test our email templates in an enormous, single test file of email strings. We will adjust our email test set up and separate these out per component for better maintainability and easier testing.
+
+## Decision outcome 3: concat `fxa-react` FTL files to other packages, use a global branding file
+
+When we use components from `fxa-react` in another package, we have to add the FTL ID and string into the package using the component rather than in `fxa-react`. This can be easily forgotten or lead to duplication of IDs/messages across our packages.
+
+Instead, we can concat files from `fxa-react` into the existing React bundles. Additionally, to help with consistency and maintainability across the board, we'll create a "global" FTL file with our license and branding to be shared across our packages to be used first to build our bundles.
+
+### Other considered options
+
+We could create a new bundle for `fxa-react` similar to other packages. However, we'd need to ship the outputted file with our other packages which would create another network request and more importantly we'd have to reconfigure our Fluent set up to check for translations in two different bundles.
+
+## Decision outcome 4: Clone `fxa-content-server-l10n` once, instead of per-package
```

Comment:
I don't have a strong opinion on the method, but I'm excited not to have to clone this for every package.

Reply:
If someone feels strongly that we should try out subtrees I would be open to it, mostly because `git-subtrees-split` and `git-subtree-merge` seem interesting, but otherwise, it seems like it could be a lot of effort to essentially completely rewrite our script that we'll have to maintain either way. Cloning is straightforward 🤷‍♀️

Reply:
I initially suggested submodules, but no longer recommend this after seeing the comments in the your working document. After thoroughly working through the pros and cons, modifying the existing clone script seems the least disruptive for the team.